src/gluonts/model/wavenet/_network.py

```diff
@@ -292,8 +282,7 @@ def target_feature_embedding(
 
         """
         # (batch_size, embed_dim, sequence_length)
-        o = self.target_embed(target).swapaxes(1, 2)
-        o = F.concat(o, features, dim=1)
+        o = F.concat(target, features, dim=1)
```

Comment:
So there is no target embedding now?

Reply:
This is all taken care by the respective representation that you choose. For example, a mean scaled representation (`MeanScaling()`) will not embed target values while a binned representation (e.g. `GlobalRelativeBinning()`) will do that. This takes place here: https://github.com/steverab/gluon-ts/blob/representation_models/src/gluonts/model/wavenet/_network.py#L383-L389